lib/pbench/cli/server/shell.py

```diff
@@ -1,8 +1,23 @@
 #!/bin/env python3
 
 from pbench.server.api import create_app
+import subprocess
 
 
 def main():
     app = create_app()
-    app.run(debug=True, port=app.config["PORT"])
+    port = app.config["PORT"]
+    host = app.config["BIND_HOST"]
+    workers = app.config["WORKERS"]
+    subprocess.run(
+        [
+            "gunicorn",
+            "--workers",
+            str(workers),
+            "--pid",
+            "/run/pbench-server.gunicorn.pid",
```

Comment:
I think the convention is to have a subdirectory under `/run'  for the subsystem (although there are certainly exceptions, e.g. `sshd`), so `/run/pbench-server/gunicorn.pid`. Not sure if that requires additional configuration though.

Reply:
yeah its probably doable via service file we can make an `ExecPre` parameter to check or create the directory `/run/pbench-server`. I will try to test it